### DISCUSSION TITLE
Suddenly no more inverter data from working inverter

### DISCUSSION THREAD

**Discussion by Stefan300381** [tbnobody/OpenDTU#1861] [Q&A]
Hi,

since three days my openDTU (up and running since months) doesn't receive any more data from the hoymiles HM 1500 converter. It could be that I updated the firmware three days ago, not sure. However, in order to troubleshoot I downgraded the firmware, but still no data. The serial monitor shows, that the NFR module is connected, but no data received. 

What can I do to further troubleshoot the issue?

PS: From my Shelly 3EM I can see that the inverter works and power is generated

```
Starting OpenDTU
Initialize FS... done
Reading configuration... done
Reading PinMapping... found valid mapping done
Initialize Network... done
Setting Hostname... Configuring WiFi STA using new credentials... done
Initialize NTP... done
Initialize SunPosition... done
Initialize MqTT... done
Initialize WebApi... done
Initialize Display... done
Initialize LEDs... done
Check for default DTU serial... done
Initialize Hoymiles interface... NRF: Connection successful
  Setting radio PA level... 
  Setting DTU serial... 
  Setting poll interval... 
  Adding inverter: 116191107181 - HM1500 done
done
Switch to WiFi mode
Setting Hostname... done
Configuring WiFi STA using new credentials... done
Configuring WiFi STA static IP... done
WiFi connected
WiFi got ip: 192.168.2.40
Network connected
Connecting to MQTT...
Connected to MQTT.
Fetch inverter: 116191107181
TX RealTimeRunData Channel: 23 --> 15 91 10 71 81 80 13 10 08 80 0B 00 66 01 22 67 00 00 00 00 00 00 00 00 2C 03 69 
RX Period End
All missing
Nothing received, resend whole request
TX RealTimeRunData Channel: 40 --> 15 91 10 71 81 80 13 10 08 80 0B 00 66 01 22 67 00 00 00 00 00 00 00 00 2C 03 69 
RX Period End
All missing
Nothing received, resend whole request
TX RealTimeRunData Channel: 61 --> 15 91 10 71 81 80 13 10 08 80 0B 00 66 01 22 67 00 00 00 00 00 00 00 00 2C 03 69
RX Period End
All missing
Nothing received, resend whole request
TX RealTimeRunData Channel: 75 --> 15 91 10 71 81 80 13 10 08 80 0B 00 66 01 22 67 00 00 00 00 00 00 00 00 2C 03 69 
RX Period End
All missing
Nothing received, resend whole request
TX RealTimeRunData Channel: 3 --> 15 91 10 71 81 80 13 10 08 80 0B 00 66 01 22 67 00 00 00 00 00 00 00 00 2C 03 69 
RX Period End
All missing
```

best regards
Stefan

**Comment by dtang-microchip** [comment]
Did you have ay luck with this?  seems like the same just happened to me - with 2 HM-600-2T's that worked for a week
end then stopped

**Comment by Stefan300381** [comment]
In the end I purchased another antenna and it still didn't work. Then I noticed that I had to resolder one Pin. A bit weird, the new antenna works like a charm and the old antenna somehow blocks the ESP from connecting to Wifi. I guess it's damaged

**Comment by ghost** [comment]
look here: [#1879](https://github.com/tbnobody/OpenDTU/discussions/1879)

**Reply by dtang-microchip** [reply]
Interesting, thanks for that.  I tried enough radio's and antennas until one of them worked... fingers crossed it stays that way

**Reply by ghost** [reply]
I was so curt because English isn't my thing and I have to use Google to translate.

Actually, as a trained radio and television technician (over 30 years ago), I should know, but I've forgotten everything.
I could imagine that, even if the radio module uses a different frequency, problems could occur due to WLAN if the antenna is incorrect.

**Comment by kynerav** [comment]
so as not to open a new branch
i have the same thing from time to time. (Hoymiles HMS-2000, the distance to the module is about 2 meters)
what i noticed is just reboots don't work, turn off/on for a few minutes - also don't work
but if I leave the esp disabled for a while, (until it shows a message about time synchronization when starting), it works

also I noticed there timezone is called "Europe/Kiev" but about 2 years ago, this zone was renamed, now it is ["Europe/Kyiv"](https://www.zeitverschiebung.net/en/timezone/europe--kyiv) maybe this is the issue.

only when I see message about wrong time it starts working
